pkgs/development/python-modules/lyricwikia/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchPypi, buildPythonPackage, pytestrunner, six, beautifulsoup4, requests }:
+buildPythonPackage rec {
+  pname = "lyricwikia";
+  version = "0.1.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0l5lkvr3299x79i7skdiggp67rzgax3s00psd1zqkxfysq27jvc8";
+  };
+
+  buildInputs = [ pytestrunner ];
+  propagatedBuildInputs = [ six beautifulsoup4 requests ];
+  doCheck = false;
```

Comment:
Can you add a comment why those tests are disabled?

Reply:
Did so in the most recent commit.